go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/russross/blackfriday v1.5.2
+	github.com/russross/blackfriday/v2 v2.0.1
```

Comment:
Is there a reason for picking v2.0.1 (released over 2 years ago) instead of v2.1.0 released recently? It seems the latter fixes some issues and could be worth picking instead (see [release notes](https://github.com/russross/blackfriday/releases/tag/v2.1.0)).

Reply:
That was an oversight on my side, using `go get github.com/russross/blackfriday/v2` instead of `go get github.com/russross/blackfriday/v2@latest`. Thanks for pointing this out.